vagrant-spk

```diff
@@ -191,6 +191,72 @@ def call_vagrant_command(sandstorm_dir, *command_args):
     print("Calling {} in {}".format(" ".join(["'{}'".format(arg) for arg in command]), sandstorm_dir))
     return subprocess.check_call(command, cwd=sandstorm_dir)
 
+def ensure_working_vboxsf_in_base_box(sandstorm_dir):
+    # If the .sandstorm/Vagrantfile refers to debian/jessie64, and does not pin it to a particular
+    # version, and the user has version 8.2.2 available on their system, then we abort the
+    # "vagrant-spk up" process because it will not work.
+    def test_vagrantfile_refers_to_jessie64():
+        with open(os.path.join(sandstorm_dir, 'Vagrantfile'), 'r') as fd:
+            for line in fd:
+                if line.strip().split() == 'config.vm.box = "debian/jessie64"'.split():
+                    return True
+        return False
+    vagrantfile_refers_to_jessie64 = test_vagrantfile_refers_to_jessie64()
+
+    def test_vagrantfile_pins_version():
+        with open(os.path.join(sandstorm_dir, 'Vagrantfile'), 'r') as fd:
+            for line in fd:
+                if line.strip().startswith('config.vm.box_version '):
```

Comment:
Fixed.

It hadn't occurred to me that anyone would write

`var=value`

since

`var = value`

looks _so_ much nicer. But you're totally right here. (-: